tests/unit-fail/j041d-link1.ref

```diff
@@ -1,13 +0,0 @@
-
```

Comment:
looks like this isn't the only .ref file that is now obsolete. Can you also delete the others?